src/main/java/seedu/clinic/logic/commands/EditCommand.java

```diff
@@ -85,11 +85,31 @@
     public static final String MESSAGE_WAREHOUSE_UNCHANGED = "The edited field will results in no change to "
             + "warehouse selected. Please check your arguments again and re-enter your edit command.";
 
+    private static final String LOG_MESSAGE_RECEIVE_EDIT_SUPPLIER = "Received instructions to edit supplier";
```

Comment:
Consistency across all log messages, should we be keeping the full stop? It concerns the above files as well.